cuqi/distribution/_gamma.py

```diff
@@ -1,37 +1,79 @@
 import numpy as np
-from scipy.special import loggamma, gammainc
+import scipy.stats as sps
 from cuqi.distribution import Distribution
+from cuqi.utilities import force_ndarray
 
 class Gamma(Distribution):
+    """
+    Represents a multivariate Gamma distribution characterized by shape and rate parameters of independent random variables x_i. Each is distributed according to the PDF function
+    
+    f(x; shape, rate) = rate^shape * x^(shape-1) * exp(-rate * x) / Gamma(shape)
```

Comment:
following @jakobsj 's suggestion of adding index i, the index i can be added to the shape and rate as well

Reply:
Yes